variables.tf

```diff
@@ -17,3 +17,8 @@ variable "enable_init_gsutil_crcmod_el" {
   description = "If not false, include stdlib::init_gsutil_crcmod_el() prior to executing startup-script-custom.  Call this function from startup-script-custom to initialize gsutil as per https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod#centos-rhel-and-fedora Intended for CentOS, RHEL and Fedora systems."
   default     = "false"
 }
+
+variable "enable_get_from_bucket" {
+  description = "If not false, include stdlib::get_from_bucket() prior to executing startup-script-custom.  Requires gsutil in the PATH.  See also enable_init_gsutil_crcmod_el feature flag."
+  default     = "false"
```

Comment:
Out of curiosity, why is this feature opt-in? This seems like a handy thing to have, what's the use case for someone wanting to have this disabled?

Reply:
See #12 which was added based on input from Morgante.  The overall design philosophy is that only truly platform-independent things should be always-enabled.  Today that's logging and config loading behavior.

Platform specific behavior is to be opt-in to make it easier to maintain and support the module.  In this case get_from_bucket depends on gsutil, which depends on platform specific compilation of the crc32 library [1].

To limit scope this is only implemented today for EL systems.  The debian family will need to opt-out of EL behavior and opt-in to their own behavior.

[1] https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod